.github/workflows/build_templates.yml

```diff
@@ -47,7 +47,7 @@ jobs:
 
             - name: Install global dependencies
               run: |
-                  AZLE_VERBOSE=true npx azle install-global-dependencies --rust --wasi2ic
+                  AZLE_VERBOSE=true npx azle dev setup --rust --wasi2ic
 
             - name: Build stable template
               run: npx azle template
```

Comment:
npx azle template has been changed to npx azle dev template

Reply:
done